packages/python/plotly/recipe/meta.yaml

```
@@ -25,6 +25,7 @@ requirements:
     - setuptools
   run:
     - python
+    - narwhals >=1.13.3
```

```suggestion
    - narwhals >=1.15.1
```

Update `narwhals` version following merge of #4921